tensorflow/compiler/xla/service/tuple_simplifier_test.cc

```diff
@@ -211,5 +217,76 @@ TEST_F(TupleSimplifierTest, IncompatibleTuples) {
   EXPECT_THAT(computation->root_instruction(), tuple);
 }
 
+TEST_F(TupleSimplifierTest, CanExcludeEntryComputation) {
```

Comment:
No need to change it, how you have it here is totally fine, but FYI most of us enjoy writing tests using the textual format.  (Update your tree and grep for `ParseHloString`.)

Reply:
i have some tests which use that text string processing thing, but I'm not sure how you can refer to the pre-optimized instructions at post-optimization check time.   

i suppose you could walk up the tree from the root and check the opcode or metadata.

Reply:
> i suppose you could walk up the tree from the root and check the opcode or metadata.

Yes, see hlo_matchers.h.